Main.cpp

```diff
@@ -191,43 +239,63 @@ int main() {
       // canBus.printRxAll();
     }
 
-    chThdSleepMilliseconds(50);
+    chThdSleepMilliseconds(10000);
   }
 }
 
-// module utility functions
 // @brief Convert bytes in RX buffer from ADC reading to temperature
 // @return temp Temperature in degrees C, resolution of .25
+// TODO: Add in per-channel moving average
 uint8_t adc_to_temp(uint8_t* rxbuf) {
   static uint16_t raw{};
-  static int norm{};
+  static double voltage{};
```

Comment:
Yea they are. I was under the impression that this was the preferred intialization method when not initializing to a literal.